localstack/services/generic_proxy.py

```diff
@@ -364,7 +364,7 @@ def get_cert_pem_file_path():
     return os.path.join(config.TMP_FOLDER, SERVER_CERT_PEM_FILE)
 
 
-def start_proxy_server(port, forward_url=None, use_ssl=None, update_listener=None,
+def start_proxy_server(port, bind_address='0.0.0.0', forward_url=None, use_ssl=None, update_listener=None,
```

Comment:
Perhaps we can also use a constant here, instead of hardcoding `'0.0.0.0'`?

Btw, we have the following defined in `constants.py` - probably needs to be renamed or generally reconsidered.
```
# host to bind to when starting the services
BIND_HOST = '0.0.0.0'
```